priv/templates/bin

```diff
@@ -69,16 +69,25 @@ ERTS_LIB_DIR="$ERTS_DIR/../lib"
 [ -f "$REL_DIR/$REL_NAME.boot" ] && BOOTFILE="$REL_NAME" || BOOTFILE=start
 cd "$ROOTDIR"
 
+{{! Define alternative field separator using ASCII US (unit separator). US is
+    not expected to appear in any argument. }}
+IFS_NORM="$IFS"
+IFS_ARGS="$(printf '\x1f')"
```

Comment:
Technically, any obscure enough ASCII code (e.g. backspace) would suffice; the choice of unit "separator" seemed to communicate more intent to use as, well, a separator!